avm/res/healthcare-apis/workspace/fhirservice/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.33.93.31351",
-      "templateHash": "14990235238154710931"
+      "version": "0.33.13.18514",
```

Comment:
downgrading

Reply:
Right, there's not really a way around it for me unfortunately. The latest version has some bugs when it comes to the AKS service. Hence I'm not using the latest one. The good news is that it has no technical impact :) (i.e., there is no new fancy feature that doesn't work with 0.33.13)